tests/test_util.py

```diff
@@ -45,3 +46,16 @@ def test_warn(capsys):
     util.warn('test warn')
     _, err = capsys.readouterr()
     assert '!!! WARNING: test warn' in err
+
+
+def test_import_app():
+    object_app = "support"
+    assert util.import_app(object_app + ":app")
+
+    with pytest.raises(ImportError) as err:
+        util.import_app("a:app")
+    assert "No module" in str(err)
+
+    with pytest.raises(AppImportError) as err:
+        util.import_app(object_app + ":wrong_app")
+    assert "Failed to find application object 'wrong_app' in '{}'".format(object_app) in str(err)
```

Comment:
Style nit: Can you move this to a variable:

```py
msg = '...'

assert msg.format(...) in str(err)
```

Reply:
Ok